IBAnimatable/GradientType.swift

```diff
@@ -6,143 +6,143 @@
 import Foundation
 
 public enum GradientType: String {
-  case SunnyDays
```

Comment:
One thing I was pointing out to @JakeLin was that you have some instances where things are not "proper" camel case. Ie: BacktotheFuture I changed to backToTheFuture, GreenandBlue, BetweenNightandDay, Curiosityblue, etc.